README.md

```diff
@@ -21,6 +22,21 @@ Open your terminal, navigate to your working directory, use ```git clone``` to g
 $ git clone https://github.com/YOUR-USERNAME/arcgis-runtime-samples-qt.git
 ```
 
+#### Clone the toolkit repo
+
+Change directory into your locally cloned samples repo and then use `git clone` to get a copy of the [ArcGIS Runtime Toolkit - Qt](https://github.com/Esri/arcgis-runtime-toolkit-qt.git).
+
+```
+# Change directory to the clone of the samples repository
+# Clone the toolkit repository into the current directory in terminal
+$ cd /arcgis-runtime-samples-qt
+$ git clone https://github.com/Esri/arcgis-runtime-toolkit-qt.git
+```
+
+Cloning the toolkit in this location will allow for the samples to automatically pick it up. If you wish to place the toolkit in another location, you will need to update the samples project file accordingly to locate the necessary .pri file.
+
+* Note - if you want to contribute to the toolkit repository as well, you will need to follow the fork step, [Fork the repo](#fork-the-repo), from above but for the toolkit instead.
```

Comment:
> [Fork the repo](#fork-the-repo), from above but for the toolkit instead.

I get what you mean, but this is a bit clunky. I'd maybe just leave this detail out - most people won't want to contribute back to the project. The toolkit repo has instructions on how to do it, so I think we can leave it out from here.